velox/exec/tests/utils/QueryAssertions.cpp

```diff
@@ -190,7 +190,9 @@ velox::variant variantAt<TypeKind::DATE>(
 
 template <TypeKind kind>
 velox::variant variantAt(const ::duckdb::Value& value) {
-  using T = typename KindToFlatVector<kind>::WrapperType;
```

Comment:
Curious, which type required this change.

Reply:
The wrapped type of VARCHAR is StringView and duckdb's ::duckdb::Value::GetValue doesn't support that template type.